docs/build.md

```diff
@@ -11,6 +11,7 @@ Install the following on your host system:
  * sudo
  * systemd-container
  * gpg
+ * curl
```

Comment:
Can you add a small note to the commit message to explain why curl is being added? i.e. "We depended on curl, but never specified it."
(This is to make it clear that the commit isn't supposed to mean "We need to start depending on curl for [reason].")